src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const RESULT = {};
```

Comment:
Avoid variable namings like `RESULT`. It will decrease code readability. Consider using a name that describes the content or purpose of the variable, like `styleObject` or `cssProperties`.